com.woltlab.wcf/templates/shared_multipleLanguageInputJavascript.tpl

```diff
@@ -5,16 +5,17 @@
 				'wcf.global.button.disabledI18n': '{jslang}wcf.global.button.disabledI18n{/jslang}'
 			});
 
-			var availableLanguages = { {implode from=$availableLanguages key=languageID item=languageName}{@$languageID}: '{$languageName}'{/implode} };
-			var values = { {implode from=$i18nValues[$elementIdentifier] key=languageID item=value}'{@$languageID}': '{$value}'{/implode} };
+			var availableLanguages = { {implode from=$availableLanguages key=languageID item=languageName}{$languageID}: '{unsafe:$languageName|encodeJS}'{/implode} };
+			{* `$value` already encoded in `I18nHandler::assignVariables()` *}
```

Comment:
And that is a bad assumption, because `WoltLabSuite/Core/Language/Input` (and `Text`) unescape the HTML. While it is incorrect to escape the value here, not escaping it makes it worse. Arguably, this piece of code has aged like fine milk.

Please return to the previous behavior which is known to be working.